src/Whoops/Exception/Frame.php

```diff
@@ -58,7 +58,9 @@ public function getFile($shortened = false)
         if ($shortened && is_string($file)) {
             // Replace the part of the path that all frames have in common, and add 'soft hyphens' for smoother line-breaks.
             $dirname = dirname(dirname(dirname(dirname(dirname(dirname(__DIR__))))));
-            $file = str_replace($dirname, "&hellip;", $file);
+            if ($dirname !== '/') {
+                $file = str_replace($dirname, "&hellip;", $file);
```

Comment:
The shortened argument isn't used by whoops, I left it in case some people made a custom handler and expect that the function returns shortened paths. I don't think the repeated code harms anyone.
Anyway, how should I use the TemplateHelper::shorten from Frame::getFile? Make it a static method, or create a TemplateHelper instance inside getFile?